src/prefect/tasks/sendgrid/sendgrid.py

```diff
@@ -37,7 +34,7 @@ def __init__(
         html_content: str = None,
         category: Union[str, List[str]] = None,
         attachment_file_path: Union[str, Path] = None,
-        sendgrid_secret: str = "SENDGRID_API_KEY",
+        sendgrid_api_key: str = None,
```

Comment:
Generally, we would have to support this old argument with a deprecation cycle. Since this is a very new task I'm tempted to just make the switch. Thoughts @zangell44 ?

Reply:
Yeah this is new enough I have no problem making the switch

Reply:
Haha, I was taking a chance.
Thanks!